src/screens/TopicPageScreen/elements/NavHeader.js

```diff
@@ -32,12 +32,8 @@ const NavHeader = (props) => {
     initialData,
     opacityHeaderAnimation,
     onShareCommunity,
-    getTopicDetail,
-    setMemberCount,
-    memberCount,
     isHeaderHide,
     topicDetail,
-    setIsFollow,
     isFollow,
     hasSearch,
     searchText,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The props passed to the component have been modified. Some props like `getTopicDetail`, `setMemberCount`, `memberCount`, and `setIsFollow` have been removed, while a new prop `followType` has been added. This seems to be part of the refactoring to handle follow type at a higher level.
```diff
-    getTopicDetail,
-    setMemberCount,
-    memberCount,
-    setIsFollow,
+    followType
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->